libraries/AP_HAL_Linux/RCOutput_Bebop.cpp

```diff
@@ -283,6 +283,28 @@ void RCOutput_Bebop::_play_sound(uint8_t sound)
     _dev->get_semaphore()->give();
 }
 
+void RCOutput_Bebop::play_note(uint8_t pwm, uint16_t period, uint16_t duration)
```

Comment:
What's the reason for this to be in RCOutput rather than in the ToneAlarm itself?  Is it so the initialization is in one single place?

I think it would be reasonable to be in RCOutput if you wanted to share the push to hw to be in the same thread. But since you are not, why not take a reference to the dev object and do it in ToneAlarm itself?

Also, there's a `RCOutput_Bebop::_play_sound()` that's unused which seems rather conflicting with this message you are sending.... or is the `pwm` here not really a `pwm` value but rather and index like `sound` in `_play_sound()`?